### ISSUE TITLE
[BUG] Datovelger fungerer ikke med voiceover

### ISSUE THREAD

**Issue by rikkekantega** [AtB-AS/mittatb-app#619]
The datepicker modal doesn't work with assistive technologies. Focus is also off, and navigating out of the modal seems impossible.

**To Reproduce**
1. Navigate using screen reader
2. Click on the time input in assistant
3. Try to select date/time
4. Try to escape/close modal

**Expected behavior**
The datetime picker should be possible to operate by using keyboard/screen reader swipe navigation. User must be able to choose departure/arrival date and time.

The datetime picker must not trap keyboard or hide focus, preventing user from escaping the modal.

**Comment by hildeor** [comment]
Må sjekke om dette fortsatt er et problem

**Comment by monopoint** [comment]
Denne ser ut til å fungere som forventet på Android. 
Assitent - velger fra og til - ser søkeresultat og komponent for å redigere tidspunkter - går inn på tidspunkt-modalen, endrer dato og tidspunkter.

**Comment by monopoint** [comment]
Denne er testet og verifisert fikset på Android og IOS og med blindeforbundet.
Saken lukkes.
